mirrord/layer/src/connection.rs

```diff
@@ -26,6 +32,10 @@ mirrord-layer failed while trying to establish connection with the agent pod!
 >> Check your kubernetes context match where the agent should be spawned.
```

Comment:
What does this mean?

Reply:
If I recall, this means to check if you set `use-context` to the correct cluster.